src/journal.c

```diff
@@ -116,119 +115,119 @@ static const UnkStruct_020E5B50 Unk_020E5B50[] = {
 
 int Journal_SaveSize(void)
 {
-    return sizeof(Journal) * 10;
+    return sizeof(JournalEntry) * MAX_JOURNAL_ENTRIES;
 }
 
-static void Journal_Init(Journal *journal)
+static void Journal_Init(JournalEntry *journalEntry)
 {
-    memset(journal, 0, sizeof(Journal));
+    memset(journalEntry, 0, sizeof(JournalEntry));
 }
 
-void Journal_Init10(Journal *journal)
+void Journal_Init10(JournalEntry *journalEntry)
 {
-    memset(journal, 0, sizeof(Journal) * 10);
+    memset(journalEntry, 0, sizeof(JournalEntry) * MAX_JOURNAL_ENTRIES);
 }
 
-Journal *SaveData_GetJournal(SaveData *param0)
+JournalEntry *SaveData_GetJournal(SaveData *saveData)
 {
-    return SaveData_SaveTable(param0, 18);
+    return SaveData_SaveTable(saveData, 18);
 }
 
-Journal *Journal_GetSavedPage(Journal *journal, BOOL param1)
+JournalEntry *Journal_GetSavedPage(JournalEntry *journalEntry, BOOL journalAcquired)
 {
-    RTCDate v0;
-    s32 v1;
+    RTCDate currDate;
+    s32 i;
 
-    if (param1 != 1) {
+    if (journalAcquired != TRUE) {
         return NULL;
     }
 
-    GetCurrentDate(&v0);
+    GetCurrentDate(&currDate);
 
-    if ((journal[0].unk_00.unk_00_7 != 0) && ((journal[0].unk_00.unk_00_0 != v0.year) || (journal[0].unk_00.unk_00_7 != v0.month) || (journal[0].unk_00.unk_00_14 != v0.day) || (journal[0].unk_00.unk_00_11 != v0.week))) {
-        for (v1 = 10 - 1; v1 >= 1; v1--) {
-            journal[v1] = journal[v1 - 1];
+    if ((journalEntry[0].title.month != 0) && ((journalEntry[0].title.year != currDate.year) || (journalEntry[0].title.month != currDate.month) || (journalEntry[0].title.day != currDate.day) || (journalEntry[0].title.week != currDate.week))) {
```

Comment:
**polish**: Clean up unnecessary parens and split conditionals up onto separate lines for legibility.